runners/direct-java/build.gradle

```diff
@@ -65,6 +66,7 @@ dependencies {
   shadow project(path: ":beam-sdks-java-core", configuration: "shadow")
   shadow library.java.joda_time
   shadow library.java.slf4j_api
+  shadow library.java.slf4j_simple
```

Comment:
Why do you want to add slf4j_simple here?

You typically only add SLF4J logging backends as test / application dependencies. (e..g add it to the validatesPortableRunner configuration)

Reply:
That was left from some debugging. Cleaning it up.